src/movie-service.js

```diff
@@ -0,0 +1,42 @@
+export class MovieService {
+  constructor() {
+    this.baseUrl = "http://react-cdp-api.herokuapp.com/movies";
+  }
+
+  searchMovie(
+    searchBy,
+    search,
+    sortBy = "",
+    order = "",
+    offset = 0,
+    limit = 18
+  ) {
+    let buildedUrl = this.baseUrl;
+    buildedUrl += "?"
+    if (searchBy) {
+      buildedUrl += `&searchBy=${searchBy}`;
+    }
+    if (search) {
+      buildedUrl += `&search=${search}`;
+    }
+    if (sortBy) {
+      buildedUrl += `&sortBy=${sortBy}`;
+      if (order) {
+        buildedUrl += `&sortOrder=${order}`;
+      }
+    }
+    if (offset) {
+      buildedUrl += `&offset=${offset}`;
+    }
+    if (limit) {
+      buildedUrl += `&limit=${limit}`;
+    }
```

Comment:
I recommend you to use [`query-string`](https://www.npmjs.com/package/query-string) npm package here

Reply:
Updated. Please review.